index.bs

```diff
@@ -551,34 +642,42 @@ spec:css-syntax-3;
 
   <div algorithm="relevant credential interfaces">
     The <dfn for="CredentialRequestOptions">relevant credential interface objects</dfn> for a given
```

Comment:
https://tabatkins.github.io/bikeshed/#:~:text=If%20a%20single%20definition%20is%20%22for%22%20multiple%20things%2C%20you%20can%20provide%20a%20comma%2Dseparated%20list%20of%20values%20in%20the%20attribute.

```suggestion
    The <dfn for="CredentialRequestOptions,CredentialCreationOptions">relevant credential interface objects</dfn> for a given
```

Reply:
ah, didn't know that, thanks!